lib/playground.dart

```diff
@@ -131,7 +132,7 @@ class Playground {
       GistFile css = chooseGistFile(gist, ['styles.css', 'style.css']);
 
       context.dartSource = dart == null ? '' : dart.contents;
-      context.htmlSource = html == null ? '' : html.contents;
+      context.htmlSource = html == null ? '' : html5lib.parse(html.contents).querySelector("body").innerHtml;
```

Comment:
How will this parser react if the html is not well-formed (i.e., no `<body>`)? Can you add a TODO: here to add a bit more resiliency?